contribs/gmf/examples/xsdattributes.js

```diff
@@ -1,6 +1,5 @@
 goog.provide('gmfapp.xsdattributes');
 
-goog.require('gmf.Themes');
```

Comment:
Please cp contribs/gmf/src/.eslintrc.yaml in  contribs/gmf/examples/ to enable advanced linting. That should catch similar errors.